modules/ROOT/pages/api-sync.adoc

```diff
@@ -1,71 +1,50 @@
-The API Sync feature of Anypoint Studio enables you to:
+With API Sync, you can:
 
-* Start designing a RAML, or OAS 2.0 and 3.0 API specification in Studio and sync it with MuleSoft VCS.
-* Pull a published RAML or OAS 2.0 and 3.0 API specification from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. +
-API sync also enables you to edit the API specification you imported from Exchange, and push your changes to MuleSoft VCS.
-+
-You can use API Sync on more than one API specification to scaffold multiple flows in different Mule configuration files.
-* Monitor the API versions that you implemented in your project and update your flows based on the latest published version.
-* Pull a RAML or OAS 2.0 and 3.0 API specification from MuleSoft VCS into Studio, edit the specification offline, and pull the updates back to MuleSoft VCS.
+* Start designing a RAML, or OAS 2.0 and 3.0 API spec in Studio and sync it with API Sync.
+* Pull a published RAML or OAS 2.0 and 3.0 API spec from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. 
+* Edit the API spec you imported from Exchange, and push your changes to MuleSoft VCS.
+* Use API Sync on more than one API spec to scaffold multiple flows in different Mule configuration files.
+* Monitor the API versions that you implement in your project and update your flows based on the latest published version.
+* Pull a RAML or OAS 2.0 and 3.0 API spec from MuleSoft VCS into Studio, edit the spec offline, and pull the updates back to MuleSoft VCS.
+* Develop your Mule applications following API Lifecycle development practices.
 
-API Sync supports importing both RAML, OAS 2.0 and OAS 3.0 specifications.
+API Sync supports importing both RAML, OAS 2.0, and OAS 3.0 spec, comes bundled by default with Studio, and requires Mule 4.1.4 and later.
 
-API Sync comes bundled by default in Studio, and requires Mule 4.1.4 and later.
+API Sync can’t work properly if the RAML file exceeds 4,000 lines of text.
 
-API sync might not work properly if the RAML file exceeds 4,000 lines of text.
+== Create an API Spec in Studio
 
-Using API Sync, you can develop your Mule applications following API Lifecycle development practices.
+Start a new API spec project in Studio. See xref:create-api-specification-studio.adoc[] for more information.
 
-== Create an API Specification in Studio
+== Import an API Spec from Exchange 
 
-. Start a new API specification project in Studio.
-See xref:create-api-specification-studio.adoc[Create an API Specification Project in Studio] for more information.
-. Publish it to Exchange.
-. Import the same API specification from Exchange to your Studio project. +
-See xref:import-api-specification-exchange.adoc[Import an API Specification from Exchange] for more information.
+Import an API spec from Exchange into a new Mule project or from MuleSoft VCS into an Editable API Spec Project.
```

Comment:
Can this be lowercase: Editable API Spec Project

Reply:
Yes! Great catch! Thanks!